### PR TITLE
adding backbone-redux to migration guide and ecosystem

### PR DESCRIPTION
**Pull request by somebody32**
Hi!

I've recently open-sourced [backbone-redux](https://github.com/redbooth/backbone-redux), a set of bindings that helps to keep your BB collections and redux tree in sync. 
It is used in production heavily now and helps us to gradually migrate our legacy Backbone/Marionette app to React + Redux without full rewrite. 
So I think that it maybe be useful for others to mention this option in the migration guide.

### CODE REVIEW

**Comment by omnidan** [comment]
Thank you! :grin:

File: docs/recipes/MigratingToRedux.md [hunk 1]
```diff
@@ -27,5 +27,5 @@ Your process will look like this:
 
 ## From Backbone
 
-Sorry, you’ll need to rewrite your model layer.  
-It’s way too different!
+Backbone's model layer is way too different compared with Redux one, so one possible option is to rewrite your app's model layer from scratch.
```
Comment (gaearon) [inline docs/recipes/MigratingToRedux.md]: Please note that the original text used correct apostrophes (’).

Comment (somebody32) [inline docs/recipes/MigratingToRedux.md]: yep, good point, sorry that I've missed that

**Comment by gaearon** [comment]
@omnidan 

In future, let's first reach a consensus on what we officially recommend before merging recommendations :wink:

dcb43261f3a4a20621f807fc370303084cc12e37

**Pull request merged**